packages/cipherstash-proxy/src/config/database.rs

```diff
@@ -81,6 +81,21 @@ impl DatabaseConfig {
         })?;
         Ok(name)
     }
+
+    #[cfg(test)]
+    pub fn for_testing() -> Self {
```

Comment:
Started with  `impl Default` but a `default` implies that there is a sensible default, but there is not.

Went with`for_testing` instead, but open to ideas.